modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManager.java

```diff
@@ -100,6 +113,12 @@
         Map<GroupPartitionId, Integer> partRecoveryStates
     ) throws IgniteCheckedException;
 
+    /**
+     * Confirm that partition states are restored. This method should be called after restoring partitions using
```

Comment:
It should be noted that this method should be called only after restoring the state of all partitions in the group.